--- Pacemaker-1-0-Pacemaker-1.0.3/configure.ac.orig	2009-05-11 14:26:54.000000000 +0200
+++ Pacemaker-1-0-Pacemaker-1.0.3/configure.ac	2009-05-11 14:26:54.000000000 +0200
@@ -746,8 +746,12 @@
 	      [
 #if defined(HAVE_CURSES_H)
 #  include <curses.h>
+#elif defined(HAVE_CURSES_CURSES_H)
+#  include <curses/curses.h>
 #elif defined(HAVE_NCURSES_H)
 #  include <ncurses.h>
+#elif defined(HAVE_NCURSES_NCURSES_H)
+#  include <ncurses/ncurses.h>
 #endif
 	      ],
 	      [printw((const char *)"Test");]
