diff -dur -x '*~' pacemaker-Pacemaker-1.1.9.orig/lib/Makefile.am pacemaker-Pacemaker-1.1.9/lib/Makefile.am
--- pacemaker-Pacemaker-1.1.9.orig/lib/Makefile.am	2013-03-08 11:48:01.000000000 +0100
+++ pacemaker-Pacemaker-1.1.9/lib/Makefile.am	2013-06-21 13:53:10.000000000 +0200
@@ -39,7 +39,7 @@
 	rm -f *.pc
 
 ## Subdirectories...
-SUBDIRS	= gnu common pengine transition cib fencing services lrmd cluster
+SUBDIRS	= gnu common pengine cib transition fencing services lrmd cluster
 DIST_SUBDIRS =  $(SUBDIRS) ais
 
 if BUILD_CS_PLUGIN
diff -dur -x '*~' pacemaker-Pacemaker-1.1.9.orig/lib/pengine/Makefile.am pacemaker-Pacemaker-1.1.9/lib/pengine/Makefile.am
--- pacemaker-Pacemaker-1.1.9.orig/lib/pengine/Makefile.am	2013-03-08 11:48:01.000000000 +0100
+++ pacemaker-Pacemaker-1.1.9/lib/pengine/Makefile.am	2013-06-21 13:54:07.000000000 +0200
@@ -28,10 +28,12 @@
 
 libpe_rules_la_LDFLAGS	= -version-info 2:2:0
 libpe_rules_la_SOURCES	= rules.c common.c
+libpe_rules_la_LIBADD  = $(top_builddir)/lib/common/libcrmcommon.la
 
 libpe_status_la_LDFLAGS	= -version-info 5:0:1
 libpe_status_la_SOURCES	=  status.c unpack.c utils.c complex.c native.c group.c clone.c rules.c common.c
-libpe_status_la_LIBADD	=  @CURSESLIBS@
+libpe_status_la_LIBADD	=  @CURSESLIBS@ \
+			   $(top_builddir)/lib/common/libcrmcommon.la
 
 clean-generic:
 	rm -f *.log *.debug *~
Only in pacemaker-Pacemaker-1.1.9/lib/pengine: Makefile.am.orig
Only in pacemaker-Pacemaker-1.1.9/lib/pengine: Makefile.am.rej
diff -dur -x '*~' pacemaker-Pacemaker-1.1.9.orig/lib/services/Makefile.am pacemaker-Pacemaker-1.1.9/lib/services/Makefile.am
--- pacemaker-Pacemaker-1.1.9.orig/lib/services/Makefile.am	2013-03-08 11:48:01.000000000 +0100
+++ pacemaker-Pacemaker-1.1.9/lib/services/Makefile.am	2013-06-21 13:53:10.000000000 +0200
@@ -26,7 +26,8 @@
 libcrmservice_la_SOURCES = services.c services_linux.c
 libcrmservice_la_LDFLAGS = -version-info 1:0:0
 libcrmservice_la_CFLAGS  = $(GIO_CFLAGS)
-libcrmservice_la_LIBADD   = $(GIO_LIBS)
+libcrmservice_la_LIBADD   = $(GIO_LIBS) \
+			    $(top_builddir)/lib/common/libcrmcommon.la
 
 if BUILD_UPSTART
 libcrmservice_la_SOURCES += upstart.c
diff -dur -x '*~' pacemaker-Pacemaker-1.1.9.orig/lib/transition/Makefile.am pacemaker-Pacemaker-1.1.9/lib/transition/Makefile.am
--- pacemaker-Pacemaker-1.1.9.orig/lib/transition/Makefile.am	2013-03-08 11:48:01.000000000 +0100
+++ pacemaker-Pacemaker-1.1.9/lib/transition/Makefile.am	2013-06-21 13:53:10.000000000 +0200
@@ -29,6 +29,7 @@
 
 libtransitioner_la_LDFLAGS	= -version-info 2:0:0
 libtransitioner_la_CFLAGS	= -I$(top_builddir)
+libtransitioner_la_LIBADD	= $(top_builddir)/lib/common/libcrmcommon.la
 
 clean-generic:
 	rm -f *~
