--- pacemaker-Pacemaker-1.1.19/configure.ac.orig	2018-09-03 19:49:56.608934960 +0200
+++ pacemaker-Pacemaker-1.1.19/configure.ac	2018-09-03 19:55:53.318930891 +0200
@@ -1110,7 +1110,7 @@
 AC_DEFINE_UNQUOTED(CRM_DAEMON_DIR,"$CRM_DAEMON_DIR", Location for Pacemaker daemons)
 AC_SUBST(CRM_DAEMON_DIR)
 
-HB_DAEMON_DIR=`try_extract_header_define $GLUE_HEADER HA_LIBHBDIR $libdir/heartbeat`
+HB_DAEMON_DIR=`try_extract_header_define $GLUE_HEADER GLUE_DAEMON_DIR $libexecdir/heartbeat`
 AC_DEFINE_UNQUOTED(HB_DAEMON_DIR,"$HB_DAEMON_DIR", Location Heartbeat expects Pacemaker daemons to be in)
 AC_SUBST(HB_DAEMON_DIR)
 
