--- pacemaker-Pacemaker-1.1.11/configure.ac.orig	2014-02-14 19:38:11.914625405 +0100
+++ pacemaker-Pacemaker-1.1.11/configure.ac	2014-02-14 19:43:35.904611808 +0100
@@ -614,17 +614,21 @@
 dnl ===============================================
 dnl Libraries
 dnl ===============================================
-AC_CHECK_LIB(socket, socket)			dnl -lsocket
-AC_CHECK_LIB(c, dlopen)				dnl if dlopen is in libc...
-AC_CHECK_LIB(dl, dlopen)			dnl -ldl (for Linux)
-AC_CHECK_LIB(rt, sched_getscheduler)            dnl -lrt (for Tru64)
-AC_CHECK_LIB(gnugetopt, getopt_long)		dnl -lgnugetopt ( if available )
-AC_CHECK_LIB(pam, pam_start)			dnl -lpam (if available)
+AC_SEARCH_LIBS(socket, socket)			dnl -lsocket
+AC_SEARCH_LIBS(dlopen, [dl])			dnl if dlopen is in libc... or -ldl (for Linux)
+AC_SEARCH_LIBS(sched_getscheduler, [rt])	dnl -lrt (for Tru64)
+AC_SEARCH_LIBS(getopt_long, [gnugetopt])	dnl -lgnugetopt ( if available )
+AC_CHECK_LIB(pam, pam_start, [LIBPAM=-lpam])	dnl -lpam (if available)
+AC_SUBST([LIBPAM])
 
 AC_CHECK_FUNCS([sched_setscheduler])
 
-AC_CHECK_LIB(uuid, uuid_parse)			dnl load the library if necessary
+AC_CHECK_LIB(uuid, uuid_parse, [LIBUUID=-luuid])	dnl load the library if necessary
+AC_SUBST([LIBUUID])
+oldLIBS="$LIBS"
+LIBS="$LIBS $LIBUUID"
 AC_CHECK_FUNCS(uuid_unparse)			dnl OSX ships uuid_* as standard functions
+LIBS="$oldLIBS"
 
 AC_CHECK_HEADERS(uuid/uuid.h)
 
@@ -772,8 +776,10 @@
 else
    XML2HEAD="`$XML2CONFIG --cflags`"
    AC_MSG_RESULT($XML2HEAD)
-   AC_CHECK_LIB(xml2, xmlReadMemory)
-   AC_CHECK_LIB(xslt, xsltApplyStylesheet)
+   AC_CHECK_LIB(xml2, xmlReadMemory, [LIBXML2=-lxml2])
+   AC_CHECK_LIB(xslt, xsltApplyStylesheet, [LIBXSLT=-lxslt])
+   AC_SUBST([LIBXML2])
+   AC_SUBST([LIBXSLT])
 fi
 
 CPPFLAGS="$CPPFLAGS $XML2HEAD"
@@ -838,7 +844,8 @@
 dnl   bzip2
 dnl ========================================================================
 AC_CHECK_HEADERS(bzlib.h)
-AC_CHECK_LIB(bz2, BZ2_bzBuffToBuffCompress)
+AC_CHECK_LIB(bz2, BZ2_bzBuffToBuffCompress, [LIBBZ2=-lbz2])
+AC_SUBST([LIBBZ2])
 
 if test x$ac_cv_lib_bz2_BZ2_bzBuffToBuffCompress != xyes ; then
    AC_MSG_ERROR(BZ2 libraries not found)
@@ -1000,7 +1007,8 @@
 
 PKG_CHECK_MODULES(libqb, libqb, HAVE_libqb=1, HAVE_libqb=0)
 AC_CHECK_HEADERS(qb/qbipc_common.h)
-AC_CHECK_LIB(qb, qb_ipcs_connection_auth_set)
+AC_CHECK_LIB(qb, qb_ipcs_connection_auth_set, [LIBQB=-lqb])
+AC_SUBST([LIBQB])
 
 LIBQB_LOG=1
 PCMK_FEATURES="$PCMK_FEATURES libqb-logging libqb-ipc"
@@ -1013,7 +1021,6 @@
    AC_MSG_FAILURE(Version of libqb is too old: v0.13 or greater requried)
 fi
 
-LIBS="$LIBS $libqb_LIBS"
 
 AC_CHECK_HEADERS(heartbeat/hb_config.h)
 AC_CHECK_HEADERS(heartbeat/glue_config.h)
@@ -1040,8 +1047,10 @@
 if test $HAVE_GLUE = 1; then
    dnl On Debian, AC_CHECK_LIBS fail if a library has any unresolved symbols
    dnl So check for all the depenancies (so they're added to LIBS) before checking for -lplumb
-   AC_CHECK_LIB(pils, PILLoadPlugin)
-   AC_CHECK_LIB(plumb, G_main_add_IPC_Channel)
+   AC_CHECK_LIB(pils, PILLoadPlugin, [LIBPILS=-lpils])
+   AC_CHECK_LIB(plumb, G_main_add_IPC_Channel, [LIBPLUMB=-lplumb])
+   AC_SUBST([LIBPILS])
+   AC_SUBST([LIBPLUMB])
 fi
 
 dnl ===============================================
@@ -1545,7 +1554,7 @@
     AC_MSG_RESULT($SUPPORT_ACL)
 
     SUPPORT_ACL=1
-    AC_CHECK_LIB(qb, qb_ipcs_connection_auth_set)
+    AC_CHECK_LIB(qb, qb_ipcs_connection_auth_set, [:])
     if test $ac_cv_lib_qb_qb_ipcs_connection_auth_set != yes; then
         SUPPORT_ACL=0
     fi
@@ -1611,8 +1620,11 @@
 	GNUTLSLIBS="`$LIBGNUTLS_CONFIG --libs`";
 	AC_MSG_RESULT($GNUTLSLIBS)
 fi
-AC_CHECK_LIB(gnutls, gnutls_init)
+AC_CHECK_LIB(gnutls, gnutls_init, [GNUTLSLIBS=-lgnutls])
+oldLIBS="$LIBS"
+LIBS="$LIBS $GNUTLSLIBS"
 AC_CHECK_FUNCS(gnutls_priority_set_direct)
+LIBS="$oldLIBS"
 
 AC_SUBST(GNUTLSHEAD)
 AC_SUBST(GNUTLSLIBS)
--- pacemaker-Pacemaker-1.1.10/pengine/Makefile.am.orig	2013-07-26 02:02:31.000000000 +0200
+++ pacemaker-Pacemaker-1.1.10/pengine/Makefile.am	2013-09-08 15:19:23.022774335 +0200
@@ -65,10 +65,10 @@
 libpengine_la_SOURCES  += native.c group.c clone.c master.c graph.c
 
 libpengine_la_LIBADD    = $(top_builddir)/lib/pengine/libpe_status.la \
-	$(top_builddir)/lib/cib/libcib.la
+	$(top_builddir)/lib/cib/libcib.la $(LIBXML2)
 
 pengine_SOURCES	= main.c
-pengine_LDADD	= $(top_builddir)/lib/cib/libcib.la $(COMMONLIBS)
+pengine_LDADD	= $(top_builddir)/lib/cib/libcib.la $(COMMONLIBS) $(LIBQB)
 # libcib for get_object_root()
 #		$(top_builddir)/lib/hbclient/libhbclient.la
 
--- pacemaker-Pacemaker-1.1.11/lib/common/Makefile.am.orig	2014-02-14 19:44:12.731276928 +0100
+++ pacemaker-Pacemaker-1.1.11/lib/common/Makefile.am	2014-02-14 19:53:20.877920591 +0100
@@ -38,7 +38,7 @@
 endif
 
 libcrmcommon_la_LDFLAGS	= -version-info 5:0:2
-libcrmcommon_la_LIBADD  = @LIBADD_DL@ $(GNUTLSLIBS)
+libcrmcommon_la_LIBADD  = @LIBADD_DL@ $(LIBPLUMB) $(GNUTLSLIBS) $(LIBXML2) $(LIBXSLT) $(LIBQB) $(LIBBZ2) $(LIBUUID)
 libcrmcommon_la_SOURCES += $(top_builddir)/lib/gnu/md5.c
 
 clean-generic:
--- pacemaker-Pacemaker-1.1.10/lib/cib/Makefile.am.orig	2013-07-26 02:02:31.000000000 +0200
+++ pacemaker-Pacemaker-1.1.10/lib/cib/Makefile.am	2013-09-08 15:20:01.962772701 +0200
@@ -33,7 +33,7 @@
 endif
 
 libcib_la_LDFLAGS	= -version-info 3:1:0 -L$(top_builddir)/lib/pengine/.libs
-libcib_la_LIBADD        = $(CRYPTOLIB) $(top_builddir)/lib/pengine/libpe_rules.la $(top_builddir)/lib/common/libcrmcommon.la
+libcib_la_LIBADD        = $(CRYPTOLIB) $(top_builddir)/lib/pengine/libpe_rules.la $(top_builddir)/lib/common/libcrmcommon.la $(GNUTLSLIBS)
 libcib_la_CFLAGS	= -I$(top_srcdir)
 
 clean-generic:
--- pacemaker-Pacemaker-1.1.10/lib/pengine/Makefile.am.orig	2013-07-26 02:02:31.000000000 +0200
+++ pacemaker-Pacemaker-1.1.10/lib/pengine/Makefile.am	2013-09-08 15:20:26.939438320 +0200
@@ -32,7 +32,7 @@
 
 libpe_status_la_LDFLAGS	= -version-info 6:0:2
 libpe_status_la_SOURCES	=  status.c unpack.c utils.c complex.c native.c group.c clone.c rules.c common.c
-libpe_status_la_LIBADD	=  @CURSESLIBS@ $(top_builddir)/lib/common/libcrmcommon.la
+libpe_status_la_LIBADD	=  @CURSESLIBS@ $(top_builddir)/lib/common/libcrmcommon.la $(LIBXML2)
 
 clean-generic:
 	rm -f *.log *.debug *~
--- pacemaker-Pacemaker-1.1.10/lib/fencing/Makefile.am.orig	2013-07-26 02:02:31.000000000 +0200
+++ pacemaker-Pacemaker-1.1.10/lib/fencing/Makefile.am	2013-09-08 17:10:35.692494309 +0200
@@ -26,6 +26,6 @@
 
 libstonithd_la_SOURCES = st_client.c
 libstonithd_la_LDFLAGS = -version-info 3:0:1
-libstonithd_la_LIBADD = $(top_builddir)/lib/common/libcrmcommon.la
+libstonithd_la_LIBADD = $(top_builddir)/lib/common/libcrmcommon.la $(LIBPILS)
 
 AM_CFLAGS = $(INCLUDES)
--- pacemaker-Pacemaker-1.1.10/cib/Makefile.am.orig	2013-07-26 02:02:31.000000000 +0200
+++ pacemaker-Pacemaker-1.1.10/cib/Makefile.am	2013-09-08 17:11:22.042492364 +0200
@@ -48,7 +48,7 @@
 			callbacks.c main.c remote.c common.c
 
 cib_LDADD		= $(top_builddir)/lib/cluster/libcrmcluster.la \
-			  $(COMMONLIBS) $(CRYPTOLIB) $(CLUSTERLIBS)
+			  $(COMMONLIBS) $(CRYPTOLIB) $(CLUSTERLIBS) $(LIBPAM)
 
 cibmon_SOURCES		= cibmon.c
 cibmon_LDADD		= $(COMMONLIBS)
