--- pacemaker-Pacemaker-1.1.24/configure.ac.orig	2022-02-08 21:41:43.127983960 +0100
+++ pacemaker-Pacemaker-1.1.24/configure.ac	2022-02-08 21:45:16.809058080 +0100
@@ -47,7 +47,7 @@ dnl   - Manually edit include/crm_config
 dnl     new defines
 dnl   - Should not include HAVE_* defines
 dnl   - Safe to include anywhere
-AM_CONFIG_HEADER(include/config.h include/crm_config.h)
+AC_CONFIG_HEADERS(include/config.h include/crm_config.h)
 ALL_LINGUAS="en fr"
 
 dnl Older distros may need: AM_INIT_AUTOMAKE($PACKAGE_NAME, $PACKAGE_VERSION)
