--- pacemaker-Pacemaker-1.1.10/include/crm/cluster.h.orig	2013-07-26 02:02:31.000000000 +0200
+++ pacemaker-Pacemaker-1.1.10/include/crm/cluster.h	2013-09-07 17:08:13.369459479 +0200
@@ -195,7 +195,9 @@
 const char *get_local_node_name(void);
 char *get_node_name(uint32_t nodeid);
 
+#if SUPPORT_COROSYNC
 char *pcmk_message_common_cs(cpg_handle_t handle, uint32_t nodeid, uint32_t pid, void *msg,
                         uint32_t *kind, const char **from);
+#endif
 
 #endif
