--- ClusterLabs-pacemaker-b5b0a7b.orig/lib/cluster/corosync.c	2012-03-28 08:14:13.000000000 +0200
+++ ClusterLabs-pacemaker-b5b0a7b/lib/cluster/corosync.c	2012-07-01 14:38:57.297207127 +0200
@@ -672,7 +672,11 @@
 
   bail:
     if(ais_ipc_handle) {
+#  if CS_USES_LIBQB
         crm_err("AIS connection failed: %p", (void*)ais_ipc_handle);
+#  else
+        crm_err("AIS connection failed: %li", (long)ais_ipc_handle);
+#  endif
     }
     return FALSE;
 }
@@ -1075,11 +1079,10 @@
     int rc = -1;
     int fd = 0;
     int quorate = 0;
-    uint32_t quorum_type = 0;
 
     crm_debug("Configuring Pacemaker to obtain quorum from Corosync");
 
-    rc = quorum_initialize(&pcmk_quorum_handle, &quorum_callbacks, &quorum_type);
+    rc = quorum_initialize(&pcmk_quorum_handle, &quorum_callbacks);
     if (rc != CS_OK) {
         crm_err("Could not connect to the Quorum API: %d\n", rc);
         goto bail;
--- ClusterLabs-pacemaker-b5b0a7b.orig/configure.ac	2012-07-01 14:27:54.000000000 +0200
+++ ClusterLabs-pacemaker-b5b0a7b/configure.ac	2012-07-01 14:49:25.664207127 +0200
@@ -1182,7 +1182,7 @@
 fi
 
 
-if test $SUPPORT_CS = 1 -a x$HAVE_oldipc = x0 ; then
+if true ; then
     dnl Support for plugins was removed about the time the IPC was
     dnl moved to libqb.
     dnl The only option now is the built-in quorum API
--- ClusterLabs-pacemaker-b5b0a7b.orig/tools/ccm_epoche.c	2012-03-28 08:14:13.000000000 +0200
+++ ClusterLabs-pacemaker-b5b0a7b/tools/ccm_epoche.c	2012-07-01 16:02:03.952168127 +0200
@@ -505,7 +505,6 @@
 {
     int rc = 0;
     int quorate = 0;
-    uint32_t quorum_type = 0;
     unsigned int nodeid = 0;
     cpg_handle_t c_handle = 0;
     quorum_handle_t q_handle = 0;
@@ -521,7 +520,7 @@
 
         case 'q':
             /* Go direct to the Quorum API */
-            rc = quorum_initialize(&q_handle, NULL, &quorum_type);
+            rc = quorum_initialize(&q_handle, NULL);
             if (rc != CS_OK) {
                 crm_err("Could not connect to the Quorum API: %d\n", rc);
                 return FALSE;

