--- pacemaker-Pacemaker-1.1.22/configure.ac.orig	2020-05-01 18:26:26.206360554 +0200
+++ pacemaker-Pacemaker-1.1.22/configure.ac	2020-05-01 18:32:45.394306317 +0200
@@ -807,17 +807,21 @@
 dnl ===============================================
 dnl Libraries
 dnl ===============================================
-AC_CHECK_LIB(socket, socket)			dnl -lsocket
-AC_CHECK_LIB(c, dlopen)				dnl if dlopen is in libc...
-AC_CHECK_LIB(dl, dlopen)			dnl -ldl (for Linux)
-AC_CHECK_LIB(rt, sched_getscheduler)            dnl -lrt (for Tru64)
-AC_CHECK_LIB(gnugetopt, getopt_long)		dnl -lgnugetopt ( if available )
-AC_CHECK_LIB(pam, pam_start)			dnl -lpam (if available)
+AC_SEARCH_LIBS(socket, socket)			dnl -lsocket
+AC_SEARCH_LIBS(dlopen, [dl])			dnl if dlopen is in libc... or -ldl (for Linux)
+AC_SEARCH_LIBS(sched_getscheduler, [rt])	dnl -lrt (for Tru64)
+AC_SEARCH_LIBS(getopt_long, [gnugetopt])	dnl -lgnugetopt ( if available )
+AC_CHECK_LIB(pam, pam_start, [LIBPAM=-lpam])	dnl -lpam (if available)
+AC_SUBST([LIBPAM])
 
 AC_CHECK_FUNCS([sched_setscheduler])
 
-AC_CHECK_LIB(uuid, uuid_parse)			dnl load the library if necessary
+AC_CHECK_LIB(uuid, uuid_parse, [LIBUUID=-luuid])	dnl load the library if necessary
+AC_SUBST([LIBUUID])
+oldLIBS="$LIBS"
+LIBS="$LIBS $LIBUUID"
 AC_CHECK_FUNCS(uuid_unparse)			dnl OSX ships uuid_* as standard functions
+LIBS="$oldLIBS"
 
 AC_CHECK_HEADERS(uuid/uuid.h)
 
@@ -941,8 +945,10 @@
 else
    XML2HEAD="`$XML2CONFIG --cflags`"
    AC_MSG_RESULT($XML2HEAD)
-   AC_CHECK_LIB(xml2, xmlReadMemory)
-   AC_CHECK_LIB(xslt, xsltApplyStylesheet)
+   AC_CHECK_LIB(xml2, xmlReadMemory, [AC_DEFINE([HAVE_LIBXML2], [1], [Have libxml2])  LIBXML2=-lxml2])
+   AC_CHECK_LIB(xslt, xsltApplyStylesheet, [AC_DEFINE([HAVE_LIBXSLT], [1], [Have libxslt])  LIBXSLT=-lxslt])
+   AC_SUBST([LIBXML2])
+   AC_SUBST([LIBXSLT])
 fi
 
 CPPFLAGS="$CPPFLAGS $XML2HEAD"
@@ -988,7 +994,8 @@
 dnl   bzip2
 dnl ========================================================================
 AC_CHECK_HEADERS(bzlib.h)
-AC_CHECK_LIB(bz2, BZ2_bzBuffToBuffCompress)
+AC_CHECK_LIB(bz2, BZ2_bzBuffToBuffCompress, [LIBBZ2=-lbz2])
+AC_SUBST([LIBBZ2])
 
 if test x$ac_cv_lib_bz2_BZ2_bzBuffToBuffCompress != xyes ; then
    AC_MSG_ERROR(BZ2 libraries not found)
@@ -1158,7 +1165,6 @@
 
 PKG_CHECK_MODULES(libqb, libqb >= 0.13)
 CPPFLAGS="$libqb_CFLAGS $CPPFLAGS"
-LIBS="$libqb_LIBS $LIBS"
 AC_CHECK_HEADERS(qb/qbipc_common.h)
 AC_CHECK_LIB(qb, qb_ipcs_connection_auth_set)
 
@@ -1193,8 +1199,10 @@
 if test $HAVE_GLUE = 1; then
    dnl On Debian, AC_CHECK_LIBS fail if a library has any unresolved symbols
    dnl So check for all the dependencies (so they're added to LIBS) before checking for -lplumb
-   AC_CHECK_LIB(pils, PILLoadPlugin)
-   AC_CHECK_LIB(plumb, G_main_add_IPC_Channel)
+   AC_CHECK_LIB(pils, PILLoadPlugin, [LIBPILS=-lpils])
+   AC_CHECK_LIB(plumb, G_main_add_IPC_Channel, [LIBPLUMB=-lplumb])
+   AC_SUBST([LIBPILS])
+   AC_SUBST([LIBPLUMB])
 fi
 
 dnl ===============================================
@@ -1776,7 +1784,7 @@
     AC_MSG_RESULT($SUPPORT_ACL)
 
     SUPPORT_ACL=1
-    AC_CHECK_LIB(qb, qb_ipcs_connection_auth_set)
+    AC_CHECK_LIB(qb, qb_ipcs_connection_auth_set, [:])
     if test $ac_cv_lib_qb_qb_ipcs_connection_auth_set != yes; then
         SUPPORT_ACL=0
     fi
@@ -1842,9 +1850,12 @@
 	GNUTLSLIBS="`$LIBGNUTLS_CONFIG --libs`";
 	AC_MSG_RESULT($GNUTLSLIBS)
 fi
-AC_CHECK_LIB(gnutls, gnutls_init)
+AC_CHECK_LIB(gnutls, gnutls_init, [GNUTLSLIBS=-lgnutls])
+oldLIBS="$LIBS"
+LIBS="$LIBS $GNUTLSLIBS"
 AC_CHECK_FUNCS(gnutls_priority_set_direct)
  AC_CHECK_FUNCS([gnutls_sec_param_to_pk_bits]) dnl since 2.12.0 (2011-03-24)
+LIBS="$oldLIBS"
 
 AC_SUBST(GNUTLSHEAD)
 AC_SUBST(GNUTLSLIBS)
--- pacemaker-Pacemaker-1.1.19/pengine/Makefile.am.orig	2018-09-02 14:24:36.610144333 +0200
+++ pacemaker-Pacemaker-1.1.19/pengine/Makefile.am	2018-09-02 14:45:59.036796358 +0200
@@ -66,14 +66,14 @@
 libpengine_la_LDFLAGS	+= $(LDFLAGS_HARDENED_LIB)
 
 libpengine_la_LIBADD	= $(top_builddir)/lib/pengine/libpe_status.la \
-			$(top_builddir)/lib/cib/libcib.la
+			$(top_builddir)/lib/cib/libcib.la $(LIBXML2)
 # -L$(top_builddir)/lib/pils -lpils -export-dynamic -module -avoid-version
 libpengine_la_SOURCES	= pengine.c allocate.c notif.c utils.c constraints.c
 libpengine_la_SOURCES	+= native.c group.c clone.c master.c graph.c utilization.c container.c
 
 pengine_CFLAGS	= $(CFLAGS_HARDENED_EXE)
 pengine_LDFLAGS	= $(LDFLAGS_HARDENED_EXE)
-pengine_LDADD	= $(top_builddir)/lib/cib/libcib.la $(COMMONLIBS)
+pengine_LDADD	= $(top_builddir)/lib/cib/libcib.la $(COMMONLIBS) $(libqb_LIBS)
 # libcib for get_object_root()
 #		$(top_builddir)/lib/hbclient/libhbclient.la
 pengine_SOURCES	= main.c
--- pacemaker-Pacemaker-1.1.24/lib/common/Makefile.am.orig	2022-02-08 21:47:15.161859798 +0100
+++ pacemaker-Pacemaker-1.1.24/lib/common/Makefile.am	2022-02-08 21:49:50.566582176 +0100
@@ -40,7 +40,7 @@ libcrmcommon_la_LDFLAGS	= -version-info
 libcrmcommon_la_CFLAGS	= $(CFLAGS_HARDENED_LIB)
 libcrmcommon_la_LDFLAGS	+= $(LDFLAGS_HARDENED_LIB)
 
-libcrmcommon_la_LIBADD	= @LIBADD_DL@ $(GNUTLSLIBS)
+libcrmcommon_la_LIBADD	= @LIBADD_DL@ $(LIBPLUMB) $(GNUTLSLIBS) $(LIBXML2) $(LIBXSLT) $(libqb_LIBS) $(LIBBZ2) $(LIBUUID) -lm
 
 # Use += rather than backlashed continuation lines for parsing by bumplibs.sh
 libcrmcommon_la_SOURCES        =
--- pacemaker-Pacemaker-1.1.16/lib/cib/Makefile.am.orig	2017-04-16 07:34:31.132737427 +0200
+++ pacemaker-Pacemaker-1.1.16/lib/cib/Makefile.am	2017-04-16 07:39:18.802734141 +0200
@@ -30,7 +30,7 @@
 libcib_la_CFLAGS	= $(CFLAGS_HARDENED_LIB)
 libcib_la_LDFLAGS	+= $(LDFLAGS_HARDENED_LIB)
 
-libcib_la_LIBADD	= $(CRYPTOLIB) $(top_builddir)/lib/pengine/libpe_rules.la $(top_builddir)/lib/common/libcrmcommon.la
+libcib_la_LIBADD	= $(CRYPTOLIB) $(top_builddir)/lib/pengine/libpe_rules.la $(top_builddir)/lib/common/libcrmcommon.la $(GNUTLSLIBS)
 
 clean-generic:
 	rm -f *.log *.debug *.xml *~
--- pacemaker-Pacemaker-1.1.22/lib/pengine/Makefile.am.orig	2020-05-01 18:33:26.394084202 +0200
+++ pacemaker-Pacemaker-1.1.22/lib/pengine/Makefile.am	2020-05-01 18:38:21.532485300 +0200
@@ -36,7 +36,7 @@
 libpe_status_la_CFLAGS	= $(CFLAGS_HARDENED_LIB)
 libpe_status_la_LDFLAGS	+= $(LDFLAGS_HARDENED_LIB)
 
-libpe_status_la_LIBADD	= @CURSESLIBS@ $(top_builddir)/lib/common/libcrmcommon.la
+libpe_status_la_LIBADD	= @CURSESLIBS@ $(top_builddir)/lib/common/libcrmcommon.la $(LIBXML2)
 # Use += rather than backlashed continuation lines for parsing by bumplibs.sh
 libpe_status_la_SOURCES        =
 libpe_status_la_SOURCES        += container.c
--- pacemaker-Pacemaker-1.1.22/lib/fencing/Makefile.am.orig	2020-05-01 18:33:26.394084202 +0200
+++ pacemaker-Pacemaker-1.1.22/lib/fencing/Makefile.am	2020-05-01 18:37:57.309283195 +0200
@@ -16,6 +16,7 @@
 
 libstonithd_la_LIBADD	= $(top_builddir)/lib/common/libcrmcommon.la
 libstonithd_la_LIBADD   += $(top_builddir)/lib/services/libcrmservice.la
+libstonithd_la_LIBADD   += $(LIBPILS)
 
 libstonithd_la_SOURCES	= st_client.c st_rhcs.c
 if BUILD_LHA_SUPPORT
--- pacemaker-Pacemaker-1.1.16/cib/Makefile.am.orig	2017-04-16 07:34:31.236070760 +0200
+++ pacemaker-Pacemaker-1.1.16/cib/Makefile.am	2017-04-16 07:42:50.786065054 +0200
@@ -36,7 +36,7 @@
 cib_LDFLAGS	= $(LDFLAGS_HARDENED_EXE)
 
 cib_LDADD	= $(top_builddir)/lib/cluster/libcrmcluster.la \
-		$(COMMONLIBS) $(CRYPTOLIB) $(CLUSTERLIBS)
+		$(COMMONLIBS) $(CRYPTOLIB) $(CLUSTERLIBS) $(LIBPAM)
 
 cib_SOURCES	= io.c messages.c notify.c \
 		callbacks.c main.c remote.c common.c
--- pacemaker-Pacemaker-1.1.24/lib/transition/Makefile.am.orig	2020-12-03 16:37:50.000000000 +0100
+++ pacemaker-Pacemaker-1.1.24/lib/transition/Makefile.am	2022-02-07 17:40:24.372315923 +0100
@@ -28,7 +28,7 @@ libtransitioner_la_CPPFLAGS	= -I$(top_bu
 libtransitioner_la_CFLAGS	= $(CFLAGS_HARDENED_LIB)
 libtransitioner_la_LDFLAGS	+= $(LDFLAGS_HARDENED_LIB)
 
-libtransitioner_la_LIBADD	= $(top_builddir)/lib/common/libcrmcommon.la
+libtransitioner_la_LIBADD	= $(top_builddir)/lib/common/libcrmcommon.la $(top_builddir)/lib/lrmd/liblrmd.la
 libtransitioner_la_SOURCES	= unpack.c graph.c utils.c
 
 clean-generic:
--- pacemaker-Pacemaker-1.1.24/lib/Makefile.am.orig	2020-12-03 16:37:50.000000000 +0100
+++ pacemaker-Pacemaker-1.1.24/lib/Makefile.am	2022-02-07 17:45:16.017402612 +0100
@@ -29,7 +29,7 @@ clean-local:
 	rm -f $(PC_FILES)
 
 ## Subdirectories...
-SUBDIRS	= gnu common pengine transition cib services fencing lrmd cluster
+SUBDIRS	= gnu common pengine cib services fencing lrmd transition cluster
 DIST_SUBDIRS = $(SUBDIRS)
 
 if BUILD_CS_PLUGIN
