--- pacemaker-Pacemaker-1.1.13/Makefile.common.orig	2015-06-24 03:09:28.000000000 +0200
+++ pacemaker-Pacemaker-1.1.13/Makefile.common	2015-12-19 09:37:15.966689989 +0100
@@ -33,7 +33,7 @@
 			  -I$(top_builddir)/libltdl -I$(top_srcdir)/libltdl
 
 if BUILD_HELP
-man8_MANS		= $(sbin_PROGRAMS:%=%.8) $(sbin_SCRIPTS:%=%.8)
+man8_MANS		= $(filter-out ipmiservicelogd.8, $(sbin_PROGRAMS:%=%.8) $(sbin_SCRIPTS:%=%.8))
 endif
 
 %.8:	% $(MAN8DEPS)
