--- pacemaker-Pacemaker-1.1.24/tools/notifyServicelogEvent.c.orig	2020-12-03 16:37:50.000000000 +0100
+++ pacemaker-Pacemaker-1.1.24/tools/notifyServicelogEvent.c	2022-02-08 21:46:52.110664895 +0100
@@ -23,13 +23,13 @@
 #include <crm/common/util.h>
 #include <crm/attrd.h>
 
-typedef enum { STATUS_GREEN = 1, STATUS_YELLOW, STATUS_RED } STATUS;
+typedef enum { STATUS_GREEN = 1, STATUS_YELLOW, STATUS_RED } STATUS_T;
 
-const char *status2char(STATUS status);
-STATUS event2status(struct sl_event *event);
+const char *status2char(STATUS_T status);
+STATUS_T event2status(struct sl_event *event);
 
 const char *
-status2char(STATUS status)
+status2char(STATUS_T status)
 {
     switch (status) {
         default:
@@ -42,10 +42,10 @@ status2char(STATUS status)
     }
 }
 
-STATUS
+STATUS_T
 event2status(struct sl_event * event)
 {
-    STATUS status = STATUS_GREEN;
+    STATUS_T status = STATUS_GREEN;
 
     crm_debug("Severity = %d, Disposition = %d", event->severity, event->disposition);
 
@@ -145,7 +145,7 @@ main(int argc, char *argv[])
     }
 
     if (rc == 0) {
-        STATUS status = STATUS_GREEN;
+        STATUS_T status = STATUS_GREEN;
         const char *health_component = "#health-ipmi";
         const char *health_status = NULL;
 
