--- pacemaker-Pacemaker-1.1.24/configure.ac.orig	2022-02-07 17:25:42.088523345 +0100
+++ pacemaker-Pacemaker-1.1.24/configure.ac	2022-02-07 17:31:38.959923340 +0100
@@ -1736,7 +1736,11 @@ else
    fi
 
    if test $SUPPORT_ESMTP != no; then
-	AC_PATH_PROGS(ESMTPCONFIG, libesmtp-config)
+	if $PKG_CONFIG --exists libesmtp-1.0 ; then
+		ESMTPCONFIG="$PKG_CONFIG libesmtp-1.0"
+	else
+		AC_PATH_PROGS(ESMTPCONFIG, libesmtp-config)
+	fi
 	if test "X${ESMTPCONFIG}" = "X"; then
 		AC_MSG_RESULT(You need the libesmtp development package to continue.)
 		SUPPORT_ESMTP=no
