--- pacemaker-Pacemaker-2.1.5/lib/pengine/Makefile.am.orig	2022-12-08 16:47:31.000000000 +0100
+++ pacemaker-Pacemaker-2.1.5/lib/pengine/Makefile.am	2023-03-03 22:39:16.173220067 +0100
@@ -34,7 +34,7 @@ libpe_status_la_LDFLAGS	= -version-info
 libpe_status_la_CFLAGS	= $(CFLAGS_HARDENED_LIB)
 libpe_status_la_LDFLAGS	+= $(LDFLAGS_HARDENED_LIB)
 
-libpe_status_la_LIBADD	= $(top_builddir)/lib/common/libcrmcommon.la
+libpe_status_la_LIBADD	= $(top_builddir)/lib/common/libcrmcommon.la $(top_builddir)/lib/cib/libcib.la
 # Use += rather than backlashed continuation lines for parsing by bumplibs
 libpe_status_la_SOURCES	=
 libpe_status_la_SOURCES	+= bundle.c
@@ -77,5 +77,8 @@ libpe_status_test_la_CFLAGS = $(libpe_st
 			      -fno-builtin -fno-inline
 libpe_status_test_la_LIBADD = $(top_builddir)/lib/common/libcrmcommon_test.la -lcmocka -lm
 
+$(top_builddir)/lib/cib/libcib.la : libpe_rules.la
+	$(MAKE) -C $(top_builddir)/lib/cib
+
 clean-generic:
 	rm -f *.log *.debug *~
